bin.src/generateCoefficients.py

```diff
@@ -6,6 +6,7 @@
 from lsst.sims.movingObjects import Orbits
 from lsst.sims.movingObjects import ChebyFits
 
+
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(description="Generate Chebyshev polynomial coefficients" +
```

Comment:
One feature I ended up adding to facilitate use on Cori was the ability to specify which objects from the orbit file to simulate.

Example: the S3M main belt asteroids we are using come in 14 orbit files, each containing 1 million objects.  Simulating 1 million objects at a time would take too long (i.e. it would violate NERSC's limits on the amount of time any one job is allowed to take).  Parsing the 14 files into several thousand smaller files seemed like more effort than I wanted to put in.  I therefore added a set of command line arguments that allowed me to specify "only simulate object numbers X through Y in the specified orbit file" so that I could start up several thousand smaller jobs that all depended on the same input file on Cori.

I'm not sure if this is a feature you want to add here, or not, but I have certainly found it useful.

This would involve adding additional information to the default output file name in order to indicate which objects are included.